Sources/RswiftCore/Generators/ColorStructGenerator.swift

```diff
@@ -48,7 +48,7 @@ struct ColorStructGenerator: ExternalOnlyStructGenerator {
     }
 
     return Struct(
-      availables: ["tvOS 11.0, *", "iOS 11.0, *"],
+      availables: [],
```

Comment:
Gebruiken we deze nu nog ergens anders? Anders kan ie er mss ook wel uit.

Reply:
No, we're currently not using the available for structs anywhere. So they can be removed.